### PR TITLE
getAreaPentagon

### LINKED ISSUE #3: get pentagon area
only apothem is given to calculate the area

### PR DESCRIPTION
**Pull request by ElaRCamo**
this resolves issue #3

### CODE REVIEW

**Review by GaelSM — COMMENTED** [review]
Funciona perfectamente

**Review by GaelSM — APPROVED** [review]
Todo bien

File: myLibrary.py [hunk 1]
```diff
@@ -19,3 +19,7 @@ def gerPerimeterCircle(radius):
 
 def gerPerimeterTriangle(side1, side2, side3):
     return side1 + side2 + side3
+
+
+def getAreaPentagon(side, aphotem):
+    return (5 * side * aphotem) / 2
```
Comment (alx3416) [inline myLibrary.py:25]:
```suggestion
    return (5 * side * aphotem) / 2
    
```
just a minor style issue

**Review by alx3416 — COMMENTED** [review]
just a minor suggestion

**Review by ZequiVitaAdMortem — APPROVED** [review]

**Review by unbreakrermario — APPROVED** [review]
me parece correcto

**Pull request merged**